src/library-authoring/backup-restore/data/api.test.ts

```diff
@@ -0,0 +1,65 @@
+// We will load the actual platform module to mutate its exported config object without mocking getConfig itself.
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createLibraryBackup, getLibraryBackupStatus } from './api';
```

Comment:
This comment on line 1 seems to be talking about `import { getConfig } from '@edx/frontend-platform';` but this is a totally different import.